src/parser.js

```diff
@@ -246,16 +247,29 @@ function createCreateStatementParser (isStrict) {
         acceptTokens: [
           { type: 'keyword', value: 'TABLE' },
           { type: 'keyword', value: 'DATABASE' },
+          { type: 'keyword', value: 'TRIGGER' },
         ],
       },
       add: (token) => {
         statement.type = `CREATE_${token.value.toUpperCase()}`;
       },
       postCanGoToNext: () => true,
     },
+    // {
+    //   preCanGoToNext: () => false,
+    //   validation: {
+    //     acceptTokens: [
+    //       { type: 'keyword', value: 'END' },
+    //     ],
+    //     add: (token) => {
+    //       statement.endStatement = 'END'
+    //     }
+    //   },
+    //   postCanGoToNext: () => false,
+    // },
```

Comment:
Is there a plan to use this commented out code in the future, or is it leftover from earlier efforts to fix this?

Reply:
From earlier efforts. I'll kill this commented code. I was more making this a PR to get some other eyes on it (not to suggest it was final)